atomkraft/test/trace.py

```diff
@@ -55,4 +56,29 @@ def test_trace(trace: PathLike, reactor: PathLike, keypath: str):
             )
         )
     print(f"Executing {test_name} ...")
-    pytest.main(["-s", test_path])
+
+    report_dir = root / "reports" / test_name
+    report_dir.mkdir(parents=True)
+
+    logging_file = report_dir / "log.txt"
```

Comment:
`logging_file` is where `logging` data (e.g. `logging.info("something happened")`) is stored.